ta4j-core/src/main/java/org/ta4j/core/num/NaN.java

```diff
@@ -43,182 +43,192 @@
-    }
-
-    @Override
-    public long longValue() {
-        throw new UnsupportedOperationException("No NaN represantation for long");
-    }
-
-    @Override
-    public float floatValue() {
-        return Float.NaN;
-    }
-
-    @Override
-    public double doubleValue() {
-        return Double.NaN;
-    }
-
-    @Override
-    public Number getDelegate() {
-        return null;
-    }
-
-    @Override
-    public String getName() {
-        return toString();
-    }
-
-    @Override
-    public String toString() {
-        return "NaN";
-    }
-
-    @Override
-    public Num plus(Num augend) {
-        return this;
-    }
-
-    @Override
-    public Num minus(Num subtrahend) {
-        return this;
-    }
-
-    @Override
-    public Num multipliedBy(Num multiplicand) {
-        return this;
-    }
-
-    @Override
-    public Num dividedBy(Num divisor) {
-        return this;
-    }
-
-    @Override
-    public Num remainder(Num divisor) {
-        return this;
-    }
-
-    @Override
-    public Num pow(int n) {
-        return this;
-    }
-
-    @Override
-    public Num pow(Num n) {
-        return this;
-    }
-
-    @Override
-    public Num log() {
-        return this;
-    }
-
-    @Override
-    public Num sqrt() {
-        return this;
-    }
-
-    @Override
-    public Num sqrt(int precision) {
-        return this;
-    }
-
-    @Override
-    public Num abs() {
-        return this;
-    }
-
-    @Override
-    public boolean isZero() {
-        return false;
-    }
-
-    @Override
-    public boolean isPositive() {
-        return false;
-    }
-
-    @Override
-    public boolean isPositiveOrZero() {
-        return false;
-    }
-
-    @Override
-    public boolean isNegative() {
-        return false;
-    }
-
-    @Override
-    public boolean isNegativeOrZero() {
-        return false;
-    }
-
-    /**
-     * NaN.isEqual(NaN) -> true
-     * 
-     * @param other the other value, not null
-     * @return flase if both values are not NaN
-     */
-    @Override
-    public boolean isEqual(Num other) {
-        return other != null && other.equals(NaN);
-    }
-
-    @Override
-    public boolean isGreaterThan(Num other) {
-        return false;
-    }
-
-    @Override
-    public boolean isGreaterThanOrEqual(Num other) {
-        return false;
-    }
-
-    @Override
-    public boolean isLessThan(Num other) {
-        return false;
-    }
-
-    @Override
-    public boolean isLessThanOrEqual(Num other) {
-        return false;
-    }
-
-    @Override
-    public Num min(Num other) {
-        return this;
-    }
-
-    @Override
-    public Num max(Num other) {
-        return this;
-    }
-
-    @Override
-    public Function<Number, Num> function() {
-        return number -> NaN;
-    }
-
-    @Override
-    public boolean isNaN() {
-        return true;
-    }
+	private static final long serialVersionUID = 9161474401436305600L;
```

Comment:
Please revert the file indentation change. You used tab for indentation, but the whole project uses 4 spaces (which is recommended).

Reply:
Issue not resolved